### PR TITLE
Update for settlement entries

### PR DESCRIPTION
**Pull request by tovebr**

### CODE REVIEW

File: Settlement/Entry/Cancel.ts [hunk 1]
```diff
@@ -1,35 +1,42 @@
+import { isoly } from "isoly"
 import { isly } from "isly"
 import { Amount } from "../../Amount"
 import { Authorization } from "../../Authorization"
+import { Identifier as SettlementIdentifier } from "../../Settlement/Identifier"
 import { Batch } from "../Batch"
 import { Fee } from "../Fee"
-
-export interface Cancel extends Cancel.Creatable {
+export interface Cancel extends Omit<Cancel.Creatable, "settlement"> {
```
Comment (SiFalorSum) [inline Settlement/Entry/Cancel.ts:8]: Add back the empty line

File: Settlement/Entry/Cancel.ts [hunk 2]
```diff
@@ -1,35 +1,42 @@
...
 export namespace Cancel {
 	export interface Creatable {
 		type: "cancel"
 		authorization?: Authorization
-		reference: string
+		reference?: string
 		batch: Batch
 		fee?: Fee
 		amount?: Amount
+		settlement: SettlementIdentifier
```
Comment (AntonCroner) [inline Settlement/Entry/Cancel.ts:23]: make it optional since older ones won't have it

**Review by AntonCroner — CHANGES_REQUESTED** [review]

File: Settlement/Entry/Cancel.ts (hunk 2 above)
Comment (tovebr) [reply Settlement/Entry/Cancel.ts:23]: Neee, we decided to go with mandatory settlement on creatable cause there are no old createables, they get turned into proper Entries and they have optional settlement

File: Settlement/Entry/Cancel.ts (hunk 1 above)
Comment (tovebr) [reply Settlement/Entry/Cancel.ts:9]: done

**Review by AntonCroner — APPROVED** [review]

**Pull request merged**